src/helpers/fix.js

```diff
@@ -128,10 +129,106 @@ export function formatRecord(record, settings = {}) {
     }
   }
 
+  function stripF884s(options) {
```

Comment:
Nearly 100 line function could be better of in own file?

Reply:
Split fix.js to several smaller files

[8e19fe5](https://github.com/NatLibFi/melinda-rest-api-commons/pull/40/commits/8e19fe5a6de1d9b82d2d1179298008b189148c18)